server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java

```diff
@@ -169,25 +170,23 @@ public TabletServerResourceManager(TabletServer tserver, VolumeManager fs) {
     long maxMemory = acuConf.getAsBytes(Property.TSERV_MAXMEM);
     boolean usingNativeMap = acuConf.getBoolean(Property.TSERV_NATIVEMAP_ENABLED) && NativeMap.isLoaded();
 
-    long blockSize = acuConf.getAsBytes(Property.TSERV_DEFAULT_BLOCKSIZE);
     long dCacheSize = acuConf.getAsBytes(Property.TSERV_DATACACHE_SIZE);
```

Comment:
Ok, so we will need to expand the current set of interfaces to add new methods with documentation.